Update Request: naps2 7.5.3 → 8.2.0

### Nixpkgs version

- Unstable (25.11)

### Package name

naps2

### Upstream version

8.2.0

### Nixpkgs version

7.5.3

### Changelog

_No response_

### Additional context

https://github.com/cyanfish/naps2/issues/577#issuecomment-3221838342

I've tried making the update myself:

<details>
<summary>package.nix</summary>

```nix
{
  lib,
  stdenv,
  buildDotnetModule,
  dotnetCorePackages,
  fetchFromGitHub,
  wrapGAppsHook3,
  gtk3,
  gdk-pixbuf,
  glib,
  sane-backends,
  libnotify,
}:

buildDotnetModule rec {
  pname = "naps2";
  version = "8.2.0";

  src = fetchFromGitHub {
    owner = "cyanfish";
    repo = "naps2";
    tag = "v${version}";
    hash = "sha256-wqal8kOSlT2ZfUeqHwsxU8SzWQVeuq52VghSFeZHesk=";
  };

  projectFile = "NAPS2.App.Gtk/NAPS2.App.Gtk.csproj";
  nugetDeps = ./deps.json;

  dotnetFlags = [
    "-p:TargetFrameworks=net9"
    "-p:EnablePreviewFeatures=true"
  ];

  executables = [ "naps2" ];

  dotnet-sdk = dotnetCorePackages.sdk_9_0;
  dotnet-runtime = dotnetCorePackages.runtime_9_0;

  nativeBuildInputs = [ wrapGAppsHook3 ];

  selfContainedBuild = true;
  runtimeDeps = [
    gtk3
    gdk-pixbuf
    glib
    sane-backends
    libnotify
  ];

  postInstall = ''
    install -D NAPS2.Setup/config/linux/com.naps2.Naps2.desktop $out/share/applications/com.naps2.Naps2.desktop
    install -D NAPS2.Lib/Icons/scanner-16-rev0.png $out/share/icons/hicolor/16x16/apps/com.naps2.Naps2.png
    install -D NAPS2.Lib/Icons/scanner-32-rev2.png $out/share/icons/hicolor/32x32/apps/com.naps2.Naps2.png
    install -D NAPS2.Lib/Icons/scanner-48-rev2.png $out/share/icons/hicolor/48x48/apps/com.naps2.Naps2.png
    install -D NAPS2.Lib/Icons/scanner-64-rev2.png $out/share/icons/hicolor/64x64/apps/com.naps2.Naps2.png
    install -D NAPS2.Lib/Icons/scanner-72-rev1.png $out/share/icons/hicolor/72x72/apps/com.naps2.Naps2.png
    install -D NAPS2.Lib/Icons/scanner-128.png $out/share/icons/hicolor/128x128/apps/com.naps2.Naps2.png
  '';

  meta = {
    description = "Scan documents to PDF and more, as simply as possible";
    homepage = "https://www.naps2.com";
    changelog = "https://github.com/cyanfish/naps2/blob/master/CHANGELOG.md";
    license = lib.licenses.gpl2Plus;
    maintainers = with lib.maintainers; [ eliandoran ];
    platforms = lib.platforms.linux;
    mainProgram = "naps2";
  };

}
```

</details>

<details>
<summary>deps.json</summary>

```json
[
  {
    "pname": "AtkSharp",
    "version": "3.24.24.95",
    "hash": "sha256-NgdWbXToBHhEVbvPrFcwXeit5iaqbBmNPQiC0jPKlnQ="
  },
  {
    "pname": "Autofac",
    "version": "8.0.0",
    "hash": "sha256-7XhDXw8hatQFjQMTIorQ5XrfDCc7EVNVyi6bGbc5fnA="
  },
  {
    "pname": "Ben.Demystifier",
    "version": "0.4.1",
    "hash": "sha256-Lkw67ask0hFtv+JoST304S8SzpM3U7nfhXLyyzfM9Os="
  },
  {
    "pname": "CairoSharp",
    "version": "3.24.24.95",
    "hash": "sha256-ycdgmQyQ1uSshI/9uMaqn5OBxRF8RADf4Tn/TptE2BU="
  },
  {
    "pname": "CommandLineParser",
    "version": "2.9.1",
    "hash": "sha256-ApU9y1yX60daSjPk3KYDBeJ7XZByKW8hse9NRZGcjeo="
  },
  {
    "pname": "EmbedIO",
    "version": "3.5.2",
    "hash": "sha256-e6GfVHXxYeUw3ntCrHokNoAS6mXArO7+vdMeUFnsSo8="
  },
  {
    "pname": "Eto.Forms",
    "version": "2.8.3",
    "hash": "sha256-bxWYDbgfhc/tFwUVAt1K9bGiixo8o58plQxRnaJzYgM="
  },
  {
    "pname": "Eto.Platform.Gtk",
    "version": "2.8.3",
    "hash": "sha256-U78xU1QwGpCEJTmAYZS5AQJU7d2SqISqZcZ10z0UYis="
  },
  {
    "pname": "GdkSharp",
    "version": "3.24.24.95",
    "hash": "sha256-NYjADgZG9TUQDIZiSSXDAxj5PyX/B7oKRo9f8Oyb4vI="
  },
  {
    "pname": "GioSharp",
    "version": "3.24.24.95",
    "hash": "sha256-5THx4af5PghPnQxXdnsC+wtVcoslh+0636WkB1FaPYg="
  },
  {
    "pname": "GLibSharp",
    "version": "3.24.24.95",
    "hash": "sha256-1pDRkKoUI9fLJBcTA2DBlpVccJl2GyAdL+VKjsFbttA="
  },
  {
    "pname": "Google.Protobuf",
    "version": "3.25.1",
    "hash": "sha256-UfP/iIcARptUcTVptj/5JQ9Jz9AG3aj+iKotsetOnH0="
  },
  {
    "pname": "Grpc.Core.Api",
    "version": "2.59.0",
    "hash": "sha256-8yARdHi5WEgINedQvMYaGfylsi2RipZ599bpBl7PUl0="
  },
  {
    "pname": "Grpc.Tools",
    "version": "2.65.0",
    "hash": "sha256-Nzpq4DIBnhZ7kX+/bwqUSMDa5NJB52iKuXARYnM5yZw="
  },
  {
    "pname": "GrpcDotNetNamedPipes",
    "version": "3.0.0",
    "hash": "sha256-Zy9ry4iZQ1QgwJ/StPgOdE1nCq0EXBe3k402ET7Tzeo="
  },
  {
    "pname": "GtkSharp",
    "version": "3.24.24.95",
    "hash": "sha256-sBvk5Ecf2i6c2fYVjMBVoXz0I6IlucOWeE2czZH9QHg="
  },
  {
    "pname": "IsExternalInit",
    "version": "1.0.3",
    "hash": "sha256-UfyGrj+hAfsvS/R7tlugTToD//ax9Fy0CameinRn1AU="
  },
  {
    "pname": "Makaretu.Dns",
    "version": "2.0.1",
    "hash": "sha256-zeIKCov4J6rKfbgsre0hEDVoFyHVGsrRAf/izZqTytA="
  },
  {
    "pname": "Microsoft.AspNetCore.App.Ref",
    "version": "6.0.36",
    "hash": "sha256-9jDkWbjw/nd8yqdzVTagCuqr6owJ/DUMi4BlUZT4hWU="
  },
  {
    "pname": "Microsoft.AspNetCore.App.Ref",
    "version": "8.0.19",
    "hash": "sha256-QySX2bih1UvwmLcn9cy1j+RuvZZwbcFKggL5Y/WcTnw="
  },
  {
    "pname": "Microsoft.AspNetCore.App.Runtime.linux-arm64",
    "version": "9.0.8",
    "hash": "sha256-lrGSct2fPz3L2nGmGUYo6IjRgFdjltEU7UqPOtl29+0="
  },
  {
    "pname": "Microsoft.Bcl.AsyncInterfaces",
    "version": "8.0.0",
    "hash": "sha256-9aWmiwMJKrKr9ohD1KSuol37y+jdDxPGJct3m2/Bknw="
  },
  {
    "pname": "Microsoft.Extensions.Configuration.Abstractions",
    "version": "8.0.0",
    "hash": "sha256-4eBpDkf7MJozTZnOwQvwcfgRKQGcNXe0K/kF+h5Rl8o="
  },
  {
    "pname": "Microsoft.Extensions.DependencyInjection",
    "version": "8.0.0",
    "hash": "sha256-+qIDR8hRzreCHNEDtUcPfVHQdurzWPo/mqviCH78+EQ="
  },
  {
    "pname": "Microsoft.Extensions.DependencyInjection.Abstractions",
    "version": "8.0.0",
    "hash": "sha256-75KzEGWjbRELczJpCiJub+ltNUMMbz5A/1KQU+5dgP8="
  },
  {
    "pname": "Microsoft.Extensions.DependencyInjection.Abstractions",
    "version": "8.0.1",
    "hash": "sha256-lzTYLpRDAi3wW9uRrkTNJtMmaYdtGJJHdBLbUKu60PM="
  },
  {
    "pname": "Microsoft.Extensions.Logging",
    "version": "8.0.0",
    "hash": "sha256-Meh0Z0X7KyOEG4l0RWBcuHHihcABcvCyfUXgasmQ91o="
  },
  {
    "pname": "Microsoft.Extensions.Logging.Abstractions",
    "version": "8.0.0",
    "hash": "sha256-Jmddjeg8U5S+iBTwRlVAVLeIHxc4yrrNgqVMOB7EjM4="
  },
  {
    "pname": "Microsoft.Extensions.Logging.Abstractions",
    "version": "8.0.1",
    "hash": "sha256-TYce3qvMr92JbAZ62ATBsocaH0joJzw0px0tYGZ9N0U="
  },
  {
    "pname": "Microsoft.Extensions.Options",
    "version": "8.0.0",
    "hash": "sha256-n2m4JSegQKUTlOsKLZUUHHKMq926eJ0w9N9G+I3FoFw="
  },
  {
    "pname": "Microsoft.Extensions.Primitives",
    "version": "8.0.0",
    "hash": "sha256-FU8qj3DR8bDdc1c+WeGZx/PCZeqqndweZM9epcpXjSo="
  },
  {
    "pname": "Microsoft.NETCore.App",
    "version": "2.1.30",
    "hash": "sha256-DucpZ4+YQ0g7GMOcEjsNJj2Ol1jynqEyUZwpzY7keYE="
  },
  {
    "pname": "Microsoft.NETCore.App.Host.linux-arm64",
    "version": "9.0.8",
    "hash": "sha256-xZHCcmlZaBG13yIQY/gBtat/ZRsHuEU32OnR36QjuCg="
  },
  {
    "pname": "Microsoft.NETCore.App.Ref",
    "version": "6.0.36",
    "hash": "sha256-9LZgVoIFF8qNyUu8kdJrYGLutMF/cL2K82HN2ywwlx8="
  },
  {
    "pname": "Microsoft.NETCore.App.Ref",
    "version": "8.0.19",
    "hash": "sha256-4ymel0R1c0HrX0plAWubJPzev52y0Fsx1esyQ1R7bXc="
  },
  {
    "pname": "Microsoft.NETCore.App.Runtime.linux-arm64",
    "version": "9.0.8",
    "hash": "sha256-H8EdL3O0ywVcBFbVnZokcMJoeLg/lII8sWOS+00n7CI="
  },
  {
    "pname": "Microsoft.NETCore.DotNetAppHost",
    "version": "2.1.30",
    "hash": "sha256-S32dRd5Qw5P/Srjaz849B0P8hcG02ZzmJcDY0GLdS2U="
  },
  {
    "pname": "Microsoft.NETCore.DotNetHostPolicy",
    "version": "2.1.30",
    "hash": "sha256-/G5C9pmkaM1ReoMTD7QNUsm+oHkjZrK9qHdrTZVUZv4="
  },
  {
    "pname": "Microsoft.NETCore.DotNetHostResolver",
    "version": "2.1.30",
    "hash": "sha256-5gx6yqwbWsWWKOmldsLnN5o8fXQf2rbSUknSEhs1c0w="
  },
  {
    "pname": "Microsoft.NETCore.Platforms",
    "version": "1.1.0",
    "hash": "sha256-FeM40ktcObQJk4nMYShB61H/E8B7tIKfl9ObJ0IOcCM="
  },
  {
    "pname": "Microsoft.NETCore.Platforms",
    "version": "1.1.1",
    "hash": "sha256-8hLiUKvy/YirCWlFwzdejD2Db3DaXhHxT7GSZx/znJg="
  },
  {
    "pname": "Microsoft.NETCore.Platforms",
    "version": "2.1.14",
    "hash": "sha256-eDbkdKujlhWqhcbembwpyM4DKrROdhJVB74YP/VjdVU="
  },
  {
    "pname": "Microsoft.NETCore.Targets",
    "version": "1.1.0",
    "hash": "sha256-0AqQ2gMS8iNlYkrD+BxtIg7cXMnr9xZHtKAuN4bjfaQ="
  },
  {
    "pname": "Microsoft.NETCore.Targets",
    "version": "1.1.3",
    "hash": "sha256-WLsf1NuUfRWyr7C7Rl9jiua9jximnVvzy6nk2D2bVRc="
  },
  {
    "pname": "Microsoft.NETCore.Targets",
    "version": "2.0.0",
    "hash": "sha256-Qk2PbbhZpPzb88JiQQcXlNK6RDBfP1of6g337RTMWVs="
  },
  {
    "pname": "Microsoft.NETFramework.ReferenceAssemblies",
    "version": "1.0.3",
    "hash": "sha256-FBoJP5DHZF0QHM0xLm9yd4HJZVQOuSpSKA+VQRpphEE="
  },
  {
    "pname": "Microsoft.NETFramework.ReferenceAssemblies.net462",
    "version": "1.0.3",
    "hash": "sha256-7mkqhFdDUAkQhV1MMwym6e+HwW4W90DkR00YcYXWbiE="
  },
  {
    "pname": "MimeKitLite",
    "version": "4.7.1",
    "hash": "sha256-vxCSs9w+xQDsUM8OiyCg6VmBi2twmqsD2BTJ1ESHLo8="
  },
  {
    "pname": "NAPS2.Mdns",
    "version": "1.0.1",
    "hash": "sha256-upKFE/ZQeDS94AFbRhKHU7FZJ1TO2aVyRVWye/MyJHY="
  },
  {
    "pname": "NAPS2.NTwain",
    "version": "1.0.1",
    "hash": "sha256-WSU/KBDz2zcnoYMH/38UsOMVB9+mOIJkqfKVzT58bGE="
  },
  {
    "pname": "NAPS2.Pdfium.Binaries",
    "version": "1.2.0",
    "hash": "sha256-wbOjJzCnTm+hN4uQEY8glfeuL1R8WEYQm1/obZ9tLmM="
  },
  {
    "pname": "NAPS2.PdfSharp",
    "version": "1.0.1",
    "hash": "sha256-HHBDTkCduzPNjwanQBvjjPP/A+kSFh0cyKVhCiXkoXQ="
  },
  {
    "pname": "NAPS2.Tesseract.Binaries",
    "version": "1.3.2",
    "hash": "sha256-gQFhYsgIZoOWtNqkq+EBoWOzjvnQP8uZadLO8bNmA1g="
  },
  {
    "pname": "NAPS2.Wia",
    "version": "2.0.3",
    "hash": "sha256-ESVuxc3k74pJkol4ZJ1BTXSOjmqXrmmB6cI7tBibX3c="
  },
  {
    "pname": "NETStandard.Library",
    "version": "2.0.0",
    "hash": "sha256-Pp7fRylai8JrE1O+9TGfIEJrAOmnWTJRLWE+qJBahK0="
  },
  {
    "pname": "NETStandard.Library",
    "version": "2.0.3",
    "hash": "sha256-Prh2RPebz/s8AzHb2sPHg3Jl8s31inv9k+Qxd293ybo="
  },
  {
    "pname": "Newtonsoft.Json",
    "version": "13.0.3",
    "hash": "sha256-hy/BieY4qxBWVVsDqqOPaLy1QobiIapkbrESm6v2PHc="
  },
  {
    "pname": "NLog",
    "version": "5.3.2",
    "hash": "sha256-b/y/IFUSe7qsSeJ8JVB0VFmJlkviFb8h934ktnn9Fgc="
  },
  {
    "pname": "NLog.Extensions.Logging",
    "version": "5.3.11",
    "hash": "sha256-DP3R51h+9kk06N63U+1C4/JCZTFiADeYTROToAA2R0g="
  },
  {
    "pname": "PangoSharp",
    "version": "3.24.24.95",
    "hash": "sha256-YhltIz1jisJqR2ZxvbYy0ybi4oGw6qR2SkjF/2aWiBQ="
  },
  {
    "pname": "Polyfill",
    "version": "4.9.0",
    "hash": "sha256-oTnmSAwMbxPFhzqBBrcSej0Nd3BKSX1kh0iwih2Iquo="
  },
  {
    "pname": "runtime.debian.8-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-EbnOqPOrAgI9eNheXLR++VnY4pHzMsEKw1dFPJ/Fl2c="
  },
  {
    "pname": "runtime.fedora.23-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-mVg02TNvJc1BuHU03q3fH3M6cMgkKaQPBxraSHl/Btg="
  },
  {
    "pname": "runtime.fedora.24-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-g9Uiikrl+M40hYe0JMlGHe/lrR0+nN05YF64wzLmBBA="
  },
  {
    "pname": "runtime.linux-arm64.Microsoft.NETCore.App",
    "version": "2.1.30",
    "hash": "sha256-0kRKwOZr5ZAoYRlSSntAZkvbG9jjIosteg79KggjOQ0="
  },
  {
    "pname": "runtime.linux-arm64.Microsoft.NETCore.DotNetAppHost",
    "version": "2.1.30",
    "hash": "sha256-gdd3w8hQ4gg+wHx0p5dc9LS+Vw1GhNa0JKfsLQ8a9QI="
  },
  {
    "pname": "runtime.linux-arm64.Microsoft.NETCore.DotNetHostPolicy",
    "version": "2.1.30",
    "hash": "sha256-3FHYUUXakKScAxdfO9U4SQPAdslGewAeRaSntzvRUr4="
  },
  {
    "pname": "runtime.linux-arm64.Microsoft.NETCore.DotNetHostResolver",
    "version": "2.1.30",
    "hash": "sha256-a8bVHitAuYoDFlBZQ5oaQR9xTTiubQTko/TplHni30s="
  },
  {
    "pname": "runtime.linux-x64.Microsoft.NETCore.App",
    "version": "2.1.30",
    "hash": "sha256-C68Sv04pdUcOcZunGCv9BD6sC+80U3Iv3FsZyZ+ayfM="
  },
  {
    "pname": "runtime.linux-x64.Microsoft.NETCore.DotNetAppHost",
    "version": "2.1.30",
    "hash": "sha256-/VWZBxBvHI8n5so2GU6Eo3Jzj4lVk+ouIH0sbyrcNFc="
  },
  {
    "pname": "runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy",
    "version": "2.1.30",
    "hash": "sha256-gaUvRbpiC8ApcwC4FA0F4G8fmvp4xDL6FrnpizmKaf8="
  },
  {
    "pname": "runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver",
    "version": "2.1.30",
    "hash": "sha256-Je0Ov7Mhqe6zH7+WTkbRkIbXbp/NefQ30xeYd6Ho3+g="
  },
  {
    "pname": "runtime.native.System",
    "version": "4.3.0",
    "hash": "sha256-ZBZaodnjvLXATWpXXakFgcy6P+gjhshFXmglrL5xD5Y="
  },
  {
    "pname": "runtime.native.System.Net.Http",
    "version": "4.3.0",
    "hash": "sha256-c556PyheRwpYhweBjSfIwEyZHnAUB8jWioyKEcp/2dg="
  },
  {
    "pname": "runtime.native.System.Security.Cryptography.Apple",
    "version": "4.3.0",
    "hash": "sha256-2IhBv0i6pTcOyr8FFIyfPEaaCHUmJZ8DYwLUwJ+5waw="
  },
  {
    "pname": "runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.0",
    "hash": "sha256-Jy01KhtcCl2wjMpZWH+X3fhHcVn+SyllWFY8zWlz/6I="
  },
  {
    "pname": "runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-xqF6LbbtpzNC9n1Ua16PnYgXHU0LvblEROTfK4vIxX8="
  },
  {
    "pname": "runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-aJBu6Frcg6webvzVcKNoUP1b462OAqReF2giTSyBzCQ="
  },
  {
    "pname": "runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-Mpt7KN2Kq51QYOEVesEjhWcCGTqWckuPf8HlQ110qLY="
  },
  {
    "pname": "runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.Apple",
    "version": "4.3.0",
    "hash": "sha256-serkd4A7F6eciPiPJtUyJyxzdAtupEcWIZQ9nptEzIM="
  },
  {
    "pname": "runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-JvMltmfVC53mCZtKDHE69G3RT6Id28hnskntP9MMP9U="
  },
  {
    "pname": "runtime.rhel.7-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-QfFxWTVRNBhN4Dm1XRbCf+soNQpy81PsZed3x6op/bI="
  },
  {
    "pname": "runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-EaJHVc9aDZ6F7ltM2JwlIuiJvqM67CKRq682iVSo+pU="
  },
  {
    "pname": "runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-PHR0+6rIjJswn89eoiWYY1DuU8u6xRJLrtjykAMuFmA="
  },
  {
    "pname": "runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl",
    "version": "4.3.2",
    "hash": "sha256-LFkh7ua7R4rI5w2KGjcHlGXLecsncCy6kDXLuy4qD/Q="
  },
  {
    "pname": "SharpZipLib",
    "version": "1.3.3",
    "hash": "sha256-HWEQTKh9Ktwg/zIl079dAiH+ob2ShWFAqLgG6XgIMr4="
  },
  {
    "pname": "SharpZipLib",
    "version": "1.4.2",
    "hash": "sha256-/giVqikworG2XKqfN9uLyjUSXr35zBuZ2FX2r8X/WUY="
  },
  {
    "pname": "SimpleBase",
    "version": "1.3.1",
    "hash": "sha256-FtRIcfLYXxUftEhq8kHgNNkahde6Fv2hPVGZNezCW1Y="
  },
  {
    "pname": "SixLabors.Fonts",
    "version": "1.0.0-beta17",
    "hash": "sha256-KnHduFn5h/mtHCpCFKg+ZIZi6fKjz2H2sqTIzwXCqOI="
  },
  {
    "pname": "SixLabors.Fonts",
    "version": "1.0.1",
    "hash": "sha256-fox6f9Z5xunVXiy71KTSkb0DirMN00tuUlChyp96kiI="
  },
  {
    "pname": "StandardSocketsHttpHandler",
    "version": "2.2.0.8",
    "hash": "sha256-0ZqwMlt7rgfYisva0DSWS5UNOd+S1NBlNWvcm+DPA6I="
  },
  {
    "pname": "System.Buffers",
    "version": "4.5.1",
    "hash": "sha256-wws90sfi9M7kuCPWkv1CEYMJtCqx9QB/kj0ymlsNaxI="
  },
  {
    "pname": "System.Collections",
    "version": "4.3.0",
    "hash": "sha256-afY7VUtD6w/5mYqrce8kQrvDIfS2GXDINDh73IjxJKc="
  },
  {
    "pname": "System.Collections.Concurrent",
    "version": "4.3.0",
    "hash": "sha256-KMY5DfJnDeIsa13DpqvyN8NkReZEMAFnlmNglVoFIXI="
  },
  {
    "pname": "System.Collections.Immutable",
    "version": "8.0.0",
    "hash": "sha256-F7OVjKNwpqbUh8lTidbqJWYi476nsq9n+6k0+QVRo3w="
  },
  {
    "pname": "System.Diagnostics.Debug",
    "version": "4.3.0",
    "hash": "sha256-fkA79SjPbSeiEcrbbUsb70u9B7wqbsdM9s1LnoKj0gM="
  },
  {
    "pname": "System.Diagnostics.DiagnosticSource",
    "version": "4.3.0",
    "hash": "sha256-OFJRb0ygep0Z3yDBLwAgM/Tkfs4JCDtsNhwDH9cd1Xw="
  },
  {
    "pname": "System.Diagnostics.DiagnosticSource",
    "version": "7.0.2",
    "hash": "sha256-8Uawe7mWOQsDzMSAAP16nuGD1FRSajyS8q+cA++MJ8E="
  },
  {
    "pname": "System.Diagnostics.Tracing",
    "version": "4.3.0",
    "hash": "sha256-hCETZpHHGVhPYvb4C0fh4zs+8zv4GPoixagkLZjpa9Q="
  },
  {
    "pname": "System.Globalization",
    "version": "4.3.0",
    "hash": "sha256-caL0pRmFSEsaoeZeWN5BTQtGrAtaQPwFi8YOZPZG5rI="
  },
  {
    "pname": "System.Globalization.Calendars",
    "version": "4.3.0",
    "hash": "sha256-uNOD0EOVFgnS2fMKvMiEtI9aOw00+Pfy/H+qucAQlPc="
  },
  {
    "pname": "System.Globalization.Extensions",
    "version": "4.3.0",
    "hash": "sha256-mmJWA27T0GRVuFP9/sj+4TrR4GJWrzNIk2PDrbr7RQk="
  },
  {
    "pname": "System.IO",
    "version": "4.3.0",
    "hash": "sha256-ruynQHekFP5wPrDiVyhNiRIXeZ/I9NpjK5pU+HPDiRY="
  },
  {
    "pname": "System.IO.FileSystem",
    "version": "4.3.0",
    "hash": "sha256-vNIYnvlayuVj0WfRfYKpDrhDptlhp1pN8CYmlVd2TXw="
  },
  {
    "pname": "System.IO.FileSystem.Primitives",
    "version": "4.3.0",
    "hash": "sha256-LMnfg8Vwavs9cMnq9nNH8IWtAtSfk0/Fy4s4Rt9r1kg="
  },
  {
    "pname": "System.Linq",
    "version": "4.3.0",
    "hash": "sha256-R5uiSL3l6a3XrXSSL6jz+q/PcyVQzEAByiuXZNSqD/A="
  },
  {
    "pname": "System.Memory",
    "version": "4.5.1",
    "hash": "sha256-7JhQNSvE6JigM1qmmhzOX3NiZ6ek82R4whQNb+FpBzg="
  },
  {
    "pname": "System.Memory",
    "version": "4.5.3",
    "hash": "sha256-Cvl7RbRbRu9qKzeRBWjavUkseT2jhZBUWV1SPipUWFk="
  },
  {
    "pname": "System.Memory",
    "version": "4.5.4",
    "hash": "sha256-3sCEfzO4gj5CYGctl9ZXQRRhwAraMQfse7yzKoRe65E="
  },
  {
    "pname": "System.Memory",
    "version": "4.5.5",
    "hash": "sha256-EPQ9o1Kin7KzGI5O3U3PUQAZTItSbk9h/i4rViN3WiI="
  },
  {
    "pname": "System.Net.Http",
    "version": "4.3.4",
    "hash": "sha256-FMoU0K7nlPLxoDju0NL21Wjlga9GpnAoQjsFhFYYt00="
  },
  {
    "pname": "System.Net.Primitives",
    "version": "4.3.0",
    "hash": "sha256-MY7Z6vOtFMbEKaLW9nOSZeAjcWpwCtdO7/W1mkGZBzE="
  },
  {
    "pname": "System.Numerics.Vectors",
    "version": "4.4.0",
    "hash": "sha256-auXQK2flL/JpnB/rEcAcUm4vYMCYMEMiWOCAlIaqu2U="
  },
  {
    "pname": "System.Numerics.Vectors",
    "version": "4.5.0",
    "hash": "sha256-qdSTIFgf2htPS+YhLGjAGiLN8igCYJnCCo6r78+Q+c8="
  },
  {
    "pname": "System.Private.Uri",
    "version": "4.3.2",
    "hash": "sha256-jB2+W3tTQ6D9XHy5sEFMAazIe1fu2jrENUO0cb48OgU="
  },
  {
    "pname": "System.Reflection",
    "version": "4.3.0",
    "hash": "sha256-NQSZRpZLvtPWDlvmMIdGxcVuyUnw92ZURo0hXsEshXY="
  },
  {
    "pname": "System.Reflection.Metadata",
    "version": "5.0.0",
    "hash": "sha256-Wo+MiqhcP9dQ6NuFGrQTw6hpbJORFwp+TBNTq2yhGp8="
  },
  {
    "pname": "System.Reflection.Primitives",
    "version": "4.3.0",
    "hash": "sha256-5ogwWB4vlQTl3jjk1xjniG2ozbFIjZTL9ug0usZQuBM="
  },
  {
    "pname": "System.Resources.Extensions",
    "version": "8.0.0",
    "hash": "sha256-5dHZdRwq0tdQanaU5Hw3QISvqSijSGrTa2VdgwifGDI="
  },
  {
    "pname": "System.Resources.ResourceManager",
    "version": "4.3.0",
    "hash": "sha256-idiOD93xbbrbwwSnD4mORA9RYi/D/U48eRUsn/WnWGo="
  },
  {
    "pname": "System.Runtime",
    "version": "4.3.0",
    "hash": "sha256-51813WXpBIsuA6fUtE5XaRQjcWdQ2/lmEokJt97u0Rg="
  },
  {
    "pname": "System.Runtime.CompilerServices.Unsafe",
    "version": "4.5.2",
    "hash": "sha256-8eUXXGWO2LL7uATMZye2iCpQOETn2jCcjUhG6coR5O8="
  },
  {
    "pname": "System.Runtime.CompilerServices.Unsafe",
    "version": "4.5.3",
    "hash": "sha256-lnZMUqRO4RYRUeSO8HSJ9yBHqFHLVbmenwHWkIU20ak="
  },
  {
    "pname": "System.Runtime.CompilerServices.Unsafe",
    "version": "4.7.0",
    "hash": "sha256-pORThFo85P8TrmfZCCPIXysVPcV2nW8hRlO6z4jVJps="
  },
  {
    "pname": "System.Runtime.CompilerServices.Unsafe",
    "version": "6.0.0",
    "hash": "sha256-bEG1PnDp7uKYz/OgLOWs3RWwQSVYm+AnPwVmAmcgp2I="
  },
  {
    "pname": "System.Runtime.Extensions",
    "version": "4.3.0",
    "hash": "sha256-wLDHmozr84v1W2zYCWYxxj0FR0JDYHSVRaRuDm0bd/o="
  },
  {
    "pname": "System.Runtime.Handles",
    "version": "4.3.0",
    "hash": "sha256-KJ5aXoGpB56Y6+iepBkdpx/AfaJDAitx4vrkLqR7gms="
  },
  {
    "pname": "System.Runtime.InteropServices",
    "version": "4.3.0",
    "hash": "sha256-8sDH+WUJfCR+7e4nfpftj/+lstEiZixWUBueR2zmHgI="
  },
  {
    "pname": "System.Runtime.Numerics",
    "version": "4.3.0",
    "hash": "sha256-P5jHCgMbgFMYiONvzmaKFeOqcAIDPu/U8bOVrNPYKqc="
  },
  {
    "pname": "System.Security.Cryptography.Algorithms",
    "version": "4.3.0",
    "hash": "sha256-tAJvNSlczYBJ3Ed24Ae27a55tq/n4D3fubNQdwcKWA8="
  },
  {
    "pname": "System.Security.Cryptography.Cng",
    "version": "4.3.0",
    "hash": "sha256-u17vy6wNhqok91SrVLno2M1EzLHZm6VMca85xbVChsw="
  },
  {
    "pname": "System.Security.Cryptography.Csp",
    "version": "4.3.0",
    "hash": "sha256-oefdTU/Z2PWU9nlat8uiRDGq/PGZoSPRgkML11pmvPQ="
  },
  {
    "pname": "System.Security.Cryptography.Encoding",
    "version": "4.3.0",
    "hash": "sha256-Yuge89N6M+NcblcvXMeyHZ6kZDfwBv3LPMDiF8HhJss="
  },
  {
    "pname": "System.Security.Cryptography.OpenSsl",
    "version": "4.3.0",
    "hash": "sha256-DL+D2sc2JrQiB4oAcUggTFyD8w3aLEjJfod5JPe+Oz4="
  },
  {
    "pname": "System.Security.Cryptography.Primitives",
    "version": "4.3.0",
    "hash": "sha256-fnFi7B3SnVj5a+BbgXnbjnGNvWrCEU6Hp/wjsjWz318="
  },
  {
    "pname": "System.Security.Cryptography.ProtectedData",
    "version": "8.0.0",
    "hash": "sha256-fb0pa9sQxN+mr0vnXg1Igbx49CaOqS+GDkTfWNboUvs="
  },
  {
    "pname": "System.Security.Cryptography.X509Certificates",
    "version": "4.3.0",
    "hash": "sha256-MG3V/owDh273GCUPsGGraNwaVpcydupl3EtPXj6TVG0="
  },
  {
    "pname": "System.Text.Encoding",
    "version": "4.3.0",
    "hash": "sha256-GctHVGLZAa/rqkBNhsBGnsiWdKyv6VDubYpGkuOkBLg="
  },
  {
    "pname": "System.Threading",
    "version": "4.3.0",
    "hash": "sha256-ZDQ3dR4pzVwmaqBg4hacZaVenQ/3yAF/uV7BXZXjiWc="
  },
  {
    "pname": "System.Threading.Tasks",
    "version": "4.3.0",
    "hash": "sha256-Z5rXfJ1EXp3G32IKZGiZ6koMjRu0n8C1NGrwpdIen4w="
  },
  {
    "pname": "System.Threading.Tasks.Dataflow",
    "version": "8.0.1",
    "hash": "sha256-hgCfF91BDd/eOtLEd5jhjzgJdvwmVv4/b42fXRr3nvo="
  },
  {
    "pname": "System.Threading.Tasks.Extensions",
    "version": "4.5.2",
    "hash": "sha256-EqJF9TppMHTKvpR6emrdA61zalMW3HwrZ7j6Bn4bBuo="
  },
  {
    "pname": "System.Threading.Tasks.Extensions",
    "version": "4.5.4",
    "hash": "sha256-owSpY8wHlsUXn5xrfYAiu847L6fAKethlvYx97Ri1ng="
  },
  {
    "pname": "System.ValueTuple",
    "version": "4.5.0",
    "hash": "sha256-niH6l2fU52vAzuBlwdQMw0OEoRS/7E1w5smBFoqSaAI="
  },
  {
    "pname": "Unosquare.Swan.Lite",
    "version": "3.1.0",
    "hash": "sha256-PL8N3CqIz/wku8/mkRMC3X868Byv47C20/rBLBhkS3o="
  },
  {
    "pname": "ZXing.Net",
    "version": "0.16.9",
    "hash": "sha256-ACVKyq6gaSYuaxA9I/GpNOlT1QPqQ6/illJycYOI8y4="
  }
]
```
</details>

<details>
<summary>But I get stuck resolving the nuget dependencies. I have a feeling it has something to do with netstandart...</summary>

```
error: builder for '/nix/store/fk6h9ixccclbvh1xkamwcdvbypyjacl7-naps2-8.2.0.drv' failed with exit code 1;
       last 25 log lines:
       > /build/source/NAPS2.Sdk/NAPS2.Sdk.csproj : error NU1101: Unable to find package runtime.any.System.Reflection. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Sdk/NAPS2.Sdk.csproj : error NU1101: Unable to find package runtime.any.System.Reflection.Primitives. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Runtime.Handles. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Sdk/NAPS2.Sdk.csproj : error NU1101: Unable to find package runtime.any.System.Globalization.Calendars. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Runtime.InteropServices. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Runtime.InteropServices. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Text.Encoding. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Threading.Tasks. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Text.Encoding. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Reflection. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Reflection.Primitives. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Threading.Tasks. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Escl/NAPS2.Escl.csproj : error NU1101: Unable to find package runtime.any.System.Globalization.Calendars. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Reflection. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Reflection.Primitives. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       > /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj : error NU1101: Unable to find package runtime.any.System.Globalization.Calendars. No packages exist with this id in source(s): _nix [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=net9]
       >   Restored /build/source/NAPS2.Images/NAPS2.Images.csproj (in 270 ms).
       >   Restored /build/source/NAPS2.Internals/NAPS2.Internals.csproj (in 270 ms).
       >   Restored /build/source/NAPS2.Images.Gtk/NAPS2.Images.Gtk.csproj (in 270 ms).
       >   Failed to restore /build/source/NAPS2.Escl/NAPS2.Escl.csproj (in 353 ms).
       >   Failed to restore /build/source/NAPS2.Escl.Server/NAPS2.Escl.Server.csproj (in 353 ms).
       >   Failed to restore /build/source/NAPS2.Sdk/NAPS2.Sdk.csproj (in 356 ms).
       >   Failed to restore /build/source/NAPS2.Lib/NAPS2.Lib.csproj (in 366 ms).
       >   Failed to restore /build/source/NAPS2.Lib.Gtk/NAPS2.Lib.Gtk.csproj (in 362 ms).
       >   Restored /build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj (in 382 ms).
       For full logs, run:
         nix log /nix/store/fk6h9ixccclbvh1xkamwcdvbypyjacl7-naps2-8.2.0.drv
```

</details>

If I enable netstandart as a target framework:

```nix
  dotnetFlags = [
    "-p:TargetFrameworks=\"net9;netstandart2.1\""
    "-p:EnablePreviewFeatures=true"
  ];
```

I get stuck at some trimming thing I know nothing about:

```
       > /nix/store/m6b2yy0lyrwdbv7g62rpr9fw0v71x5ba-dotnet-sdk-9.0.304/share/dotnet/sdk/9.0.304/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(107,5): error NETSDK1124: Trimming assemblies requires .NET Core 3.0 or higher. [/build/source/NAPS2.App.Gtk/NAPS2.App.Gtk.csproj::TargetFramework=netstandart2.1]
```

Trimming seems to be enabled for some projects but not others in naps2. This issue seems to suggest some elaborate modifications to the project configuration: https://github.com/dotnet/runtime/issues/94406#issuecomment-2070210366.

I have a feeling it may not be best to forcably overwrite the target framework for all the projects.

But I don't think I fully understand what's going on here.

### Notify maintainers

@eliandoran

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this package update does not yet exist in an [open pull request](https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+package+%28update%29%22) or in [Nixpkgs Unstable](https://search.nixos.org/packages?channel=unstable).
- [x] I assert that this is not a [duplicate of any known issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%229.needs%3A+package+%28update%29%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/na/naps2/01-donate-button.patch
+++ b/pkgs/by-name/na/naps2/01-donate-button.patch
@@ -1,0 +1,14 @@
+diff --git a/NAPS2.Lib/EtoForms/Ui/AboutForm.cs b/NAPS2.Lib/EtoForms/Ui/AboutForm.cs
+--- a/NAPS2.Lib/EtoForms/Ui/AboutForm.cs
++++ b/NAPS2.Lib/EtoForms/Ui/AboutForm.cs
+@@ -26,8 +26,8 @@ public class AboutForm : EtoDialogBase
+         _donateButton = C.Button(UiStrings.Donate, () => ProcessHelper.OpenUrl(DONATE_URL));
+         _donateButton.BackgroundColor = Color.FromRgb(0xfeda96);
+         _donateButton.TextColor = Color.FromRgb(0x1b464e);
+-        _donateButton.Font = new Font(_donateButton.Font.Family, _donateButton.Font.Size * 11 / 10,
+-            FontStyle.Italic | FontStyle.Bold);
++        try { _donateButton.Font = new Font(_donateButton.Font.Family, _donateButton.Font.Size * 11 / 10,
++            FontStyle.Italic | FontStyle.Bold); } catch {};
+         EtoPlatform.Current.ConfigureDonateButton(_donateButton);
+ 
+         _enableDebugLogging.Checked = config.Get(c => c.EnableDebugLogging);

--- a/pkgs/by-name/na/naps2/02-button-dpi.patch
+++ b/pkgs/by-name/na/naps2/02-button-dpi.patch
@@ -1,0 +1,14 @@
+diff --git a/NAPS2.Lib/EtoForms/Layout/C.cs b/NAPS2.Lib/EtoForms/Layout/C.cs
+--- a/NAPS2.Lib/EtoForms/Layout/C.cs
++++ b/NAPS2.Lib/EtoForms/Layout/C.cs
+@@ -121,8 +121,8 @@ public static class C
+         if (flags.HasFlag(ButtonFlags.LargeText))
+         {
+             var baseFontSize = button.Font.Size;
+-            EtoPlatform.Current.AttachDpiDependency(button,
+-                _ => button.Font = new Font(button.Font.Family, baseFontSize * 4 / 3));
++            EtoPlatform.Current.AttachDpiDependency(button, _ => {
++                try { button.Font = new Font(button.Font.Family, baseFontSize * 4 / 3); } catch {}});
+         }
+         EtoPlatform.Current.ConfigureImageButton(button, flags);
+         return button;

--- a/pkgs/by-name/na/naps2/package.nix
+++ b/pkgs/by-name/na/naps2/package.nix
@@ -14,32 +14,31 @@
 
 buildDotnetModule rec {
   pname = "naps2";
-  version = "7.5.3";
+  version = "8.2.1";
 
   src = fetchFromGitHub {
     owner = "cyanfish";
     repo = "naps2";
     tag = "v${version}";
-    hash = "sha256-vX+ZyCQsYqJjgYaufWJRnzX8retiFK5QHSP40bbBaCc=";
+    hash = "sha256-1OPFWmy9eDRnMJjYdzYubgfde7MNix8ZsSuN2ZHsvco=";
   };
+
+  patches = [
+    ./01-donate-button.patch
+    ./02-button-dpi.patch
+  ];
 
   projectFile = "NAPS2.App.Gtk/NAPS2.App.Gtk.csproj";
   nugetDeps = ./deps.json;
 
-  postPatch = ''
-    substituteInPlace NAPS2.Images.Gtk/NAPS2.Images.Gtk.csproj \
-      --replace-fail TargetFramework TargetFrameworks \
-  '';
-
   dotnetFlags = [
-    "-p:TargetFrameworks=net8"
-    "-p:EnablePreviewFeatures=true"
+    "-p:TargetFrameworks=net9"
   ];
 
   executables = [ "naps2" ];
 
-  dotnet-sdk = dotnetCorePackages.sdk_8_0;
-  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  dotnet-sdk = dotnetCorePackages.sdk_9_0;
+  dotnet-runtime = dotnetCorePackages.runtime_9_0;
 
   nativeBuildInputs = [ wrapGAppsHook3 ];
 
@@ -60,6 +59,14 @@ buildDotnetModule rec {
     install -D NAPS2.Lib/Icons/scanner-64-rev2.png $out/share/icons/hicolor/64x64/apps/com.naps2.Naps2.png
     install -D NAPS2.Lib/Icons/scanner-72-rev1.png $out/share/icons/hicolor/72x72/apps/com.naps2.Naps2.png
     install -D NAPS2.Lib/Icons/scanner-128.png $out/share/icons/hicolor/128x128/apps/com.naps2.Naps2.png
+    case "${stdenv.hostPlatform.system}" in
+      x86_64-linux)
+        chmod a+x $out/lib/naps2/_linux/tesseract
+        ;;
+      aarch64-linux)
+        chmod a+x $out/lib/naps2/_linuxarm/tesseract
+        ;;
+    esac
   '';
 
   meta = {

